README.md

```diff
@@ -35,7 +36,7 @@ The UI is inspired by a design on [Dribbble](https://dribbble.com/shots/10847147
 
 *Feel free to fork and contribute to include these features.* ❤︎
 
-## 🚀 Technologies
```

Comment:
I want to keep the logos

Reply:
This is just for a school assignment where I make suggestions to a README file so you can take me suggestions with a grain of salt.